### PR TITLE
Make install instructions for Docker Compose clearer

### LINKED ISSUE #2435: Compose install instructions for Linux are unclear
File: [compose/install.md](https://docs.docker.com/compose/install/)

it should be mentioned, that docker-compose can be installed with `sudo apt install docker-compose` on ubuntu. 

After reading this article (and several other) I assumed you always had to build it from source, because for some reason it isn't in the official repositories.

### PR DESCRIPTION
**Pull request by shubheksha**
### Related issues (optional)
Fixes #2435

### CODE REVIEW

**Comment by bhavin192** [comment]
I think installing `docker-compose` using `apt-get` will install some old version of compose instead of latest one.

**Comment by joaofnfernandes** [comment]
@bhavin192 is right. Currently `apt-get install` installs compose 1.5, which is pretty hold. I'm placing this PR on hold until @shin- weights on this.

**Comment by mstanleyjones** [comment]
The correct way to install Docker Compose on Linux is by following the instructions here: https://github.com/docker/compose/releases

File: compose/install.md [hunk 1]
```diff
@@ -59,6 +60,14 @@ To install Compose, do the following:
 
 ## Alternative install options
 
+### Install using apt-get on Ubuntu
+
+Compose can be installed on Ubuntu using the following command:
+
+```bash
+$ sudo apt-get install docker-compose
+```
```
Comment (dnephin) [inline compose/install.md]: I don't think we should suggest this method of install. The package is not provided by Docker, and it is not well maintained (version is version old).

**Comment by shubheksha** [comment]
Removed the section on install instructions for Ubuntu.

**Comment by johndmulhausen** [comment]
Thanks to @joaofnfernandes, @mstanleyjones, @dnephin, and @bhavin192 for the reviews. Thanks to @shubheksha for the PR!

**Pull request merged**